cmd/metalbond-speaker/main.go

```diff
@@ -50,6 +51,7 @@ func main() {
 	var vni int
 	var metalbondServer string
 	var nodeAddress string
+	var connectionTimeoutDuration = 20 * time.Second
```

Comment:
Can we call this metalbond connection timeout or something like that.

Reply:
Sure, done.